source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,347 @@ Sorts Builders
    :class: singlecol
 
 .. _sorts-builders:
+
+Overview
+--------
+
+In this guide, we show you how to specify **sort criteria** for your
+queries using **builders**.
+
+Sort criteria are the rules MongoDB uses to sort your data. Some
+examples of sort criteria are:
+
+* Smallest number to largest number
+* Earliest time of day to latest time of day
+* Alphabetical order by first name 
+
+Builders are classes provided by the MongoDB Java driver that allow you to use
+the builder pattern to interact with your MongoDB instance. The builder
+pattern allows you to create complex immutable objects separately from
+their representation. The :doc:`builders </fundamentals/builders/>`
+provided by the MongoDB Java driver can improve the readability of your code.
+
+You should read this guide if you would like to:
+
+* Use the builder pattern to specify sort criteria for your queries.
+* Learn how to perform ascending sorts and descending sorts.
+* Learn how to combine sort criteria.
+* Sort on the text score of a
+  :manual:`text search </core/text-search-operators/>`.
+
+If you want to learn the fundamentals of sorting in the MongoDB Java
+driver, consider reading our
+:doc:`guide on sorting </fundamentals/crud/read-operations/sort/>`
+for a general introduction.
+
+The Sorts Class
+---------------
+
+The :java-docs:`Sorts </apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html>`
+class is the sort criteria builder provided by the MongoDB Java Driver.
+``Sorts`` provides static factory methods for all sort criteria
+operators supported by MongoDB. All methods return a
+:java-docs:`Bson </apidocs/bson/org/bson/conversions/Bson.html>`
+object that you can pass to the 
+:java-docs:`sort() </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#sort(org.bson.conversions.Bson)>`
+method of a ``FindIterable`` instance. 
+
+.. _sorts-builders-sort-example:
+
+The following examples show you how to use the static factory methods
+provided by the ``Sorts`` class to sort your queries. The examples use a
+sample collection ``sort_example`` that contains the following documents:
+
+.. code-block:: json
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+
+Sorting Direction
+-----------------
+
+The ``Sorts`` class provides two methods for specifying the sort direction
+of your collection of data. Your data can be sorted from smallest to
```

Comment:
Thanks for the note. Good catch. The user is doing the sorting here. I rewrote the sentence so that "You can sort your data from...". Fix: https://github.com/mongodb/docs-java/pull/59/commits/4733ad5cbd7ab5fb1aba8c831244aefce5c1acaf

I will try to spot passive voice more effectively going forward. I added "look into Grammarly" to my Trello.